extra/2-piping.js

```diff
@@ -16,26 +16,30 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+    return a * b;
 }
 
-function format() {
-
+function format(value) {
+  let currency = "£"
+   return "£" + value;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
```

Comment:
Question: have we answered the question here

Reply:
If the starting value is not a number, the code may return an error.
If the add (10) or multiply (2) in functions are not defined , the code may not produce the correct result.
If the format function is not defined or doesn't behave as expected, the code may not produce the desired string representation of the result.